### ISSUE TITLE
Setting post as featured cannot be undone

### ISSUE THREAD

**Issue by MikkCZ** [mozilla/One-Mozilla-blog#23]
The issue is hidden here https://github.com/mozilla/One-Mozilla-blog/blob/master/themes/OneMozilla/functions.php#L462 - the "_fc_featuredpost" post meta is updated only if $_POST["_fc_featuredpost"] is set = only if the "_fc_featuredpost" input was successful and checked. Am I right?

If so, this seems to have a pretty simple fix, but I would like to do some testing before. Possible use case I am afraid of:
* mark post as featured
* save the post
* log in as another user (who does not have the featured widget switched on for editing posts)
* edit the post
* save the post
* is the previous "_fc_featuredpost" value send with the post edit form, or not?

**Comment by loganrosen** [comment]
The easy fix should just be to add an else clause where you unset the featured post if it is not in the POST, right?

**Comment by craigcook** [comment]
So it seems when I first wrote the function I was assuming the state of the checkbox would be saved each time the post was updated, except the unchecked box was simply a null value rather than an explicit 'false' so there was nothing to save. Hence, no way to unfeature a previously featured post. 

I've added the 'else' condition so the state of the checkbox is saved every time, where checked = true and unchecked = false. This seems to do the trick but if someone else can review/verify I'd be most appreciative.
